### ISSUE TITLE
Coroutine debugger does not work with 1.5.0-RC

### ISSUE THREAD

**Issue by elizarov** [Kotlin/kotlinx.coroutines#2690]
See https://youtrack.jetbrains.com/issue/KTIJ-11620
The error is: 

```
Exception is thrown by calling dumpCoroutines.

java.lang.IllegalStateException: Unable to find loaded class _COROUTINE._CREATION
	at org.jetbrains.kotlin.idea.debugger.coroutine.proxy.LocationCache.createLocation(LocationCache.kt:17)
...
```

**Comment by dkhalanskyjb** [comment]
`_COROUTINE._CREATION` was introduced in 1.5.0-RC: https://github.com/Kotlin/kotlinx.coroutines/pull/2461, and it is now transitively used during the initialization of `DebugProbesImpl`. This is probably the root cause for this issue, as, looking at https://github.com/JetBrains/kotlin/blob/master/idea/jvm-debugger/jvm-debugger-coroutine/src/org/jetbrains/kotlin/idea/debugger/coroutine/proxy/mirror/coroutinesDebugMirror.kt, my surface understanding is that the plugin proxies every class that could be needed for `DebugProbesImpl` to function.

Looking at the jar file for 1.5.0-RC, it does contain the `_COROUTINE._CREATION` class, so there should be no problems with loading it; it's unclear though why `ArtificialStackFrames`, a class that is also needed now for `DebugProbesImpl` to function, does not provoke the same reaction.

Also, while I am somewhat able to reproduce this, I get a different stack trace:
```
java.lang.IllegalArgumentException: Argument for @NotNull parameter 'refType' of com/intellij/debugger/engine/DebuggerUtils.findMethod must not be null
	at com.intellij.debugger.engine.DebuggerUtils.$$$reportNull$$$0(DebuggerUtils.java)
	at com.intellij.debugger.engine.DebuggerUtils.findMethod(DebuggerUtils.java)
	at com.intellij.debugger.jdi.GeneratedLocation.<init>(GeneratedLocation.java:21)
	at org.jetbrains.kotlin.idea.debugger.coroutine.proxy.LocationCache.createLocation(LocationCache.kt:36)
	at org.jetbrains.kotlin.idea.debugger.coroutine.proxy.LocationCache.createLocation(LocationCache.kt:17)
	at org.jetbrains.kotlin.idea.debugger.coroutine.proxy.CoroutineLibraryAgent2Proxy.findStackFrames(CoroutineLibraryAgent2Proxy.kt:86)
	at org.jetbrains.kotlin.idea.debugger.coroutine.proxy.CoroutineLibraryAgent2Proxy.mapToCoroutineInfoData(CoroutineLibraryAgent2Proxy.kt:37)
	at org.jetbrains.kotlin.idea.debugger.coroutine.proxy.CoroutineLibraryAgent2Proxy.dumpCoroutinesInfo(CoroutineLibraryAgent2Proxy.kt:30)
	at org.jetbrains.kotlin.idea.debugger.coroutine.proxy.CoroutineDebugProbesProxy.dumpCoroutines(CoroutineDebugProbesProxy.kt:29)
	at org.jetbrains.kotlin.idea.debugger.coroutine.view.XCoroutineView$CoroutineGroupContainer$computeChildren$1.invoke(XCoroutineView.kt:154)
	at org.jetbrains.kotlin.idea.debugger.coroutine.view.XCoroutineView$CoroutineGroupContainer$computeChildren$1.invoke(XCoroutineView.kt:147)
	at org.jetbrains.kotlin.idea.debugger.coroutine.proxy.ManagerThreadExecutor$ManagerThreadExecutorInstance$makeCommand$1.contextAction(ManagerThreadExecutor.kt:45)
	at com.intellij.debugger.engine.events.SuspendContextCommandImpl.action(SuspendContextCommandImpl.java:57)
	at com.intellij.debugger.engine.events.DebuggerCommandImpl.run(DebuggerCommandImpl.java:43)
	at com.intellij.debugger.engine.DebuggerManagerThreadImpl.processEvent(DebuggerManagerThreadImpl.java:154)
	at com.intellij.debugger.engine.DebuggerManagerThreadImpl.processEvent(DebuggerManagerThreadImpl.java:29)
	at com.intellij.debugger.impl.InvokeThread.lambda$run$0(InvokeThread.java:138)
	at com.intellij.openapi.project.DumbService.runWithAlternativeResolveEnabled(DumbService.java:367)
	at com.intellij.debugger.impl.InvokeThread.run(InvokeThread.java:125)
	at com.intellij.debugger.impl.InvokeThread$WorkerThreadRequest.lambda$run$0(InvokeThread.java:49)
	at com.intellij.util.ConcurrencyUtil.runUnderThreadName(ConcurrencyUtil.java:208)
	at com.intellij.debugger.impl.InvokeThread$WorkerThreadRequest.run(InvokeThread.java:48)
	at com.intellij.util.RunnableCallable.call(RunnableCallable.java:20)
	at com.intellij.util.RunnableCallable.call(RunnableCallable.java:11)
	at com.intellij.openapi.application.impl.ApplicationImpl$1.call(ApplicationImpl.java:270)
	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
	at java.base/java.util.concurrent.Executors$PrivilegedThreadFactory$1$1.run(Executors.java:668)
	at java.base/java.util.concurrent.Executors$PrivilegedThreadFactory$1$1.run(Executors.java:665)
	at java.base/java.security.AccessController.doPrivileged(Native Method)
	at java.base/java.util.concurrent.Executors$PrivilegedThreadFactory$1.run(Executors.java:665)
	at java.base/java.lang.Thread.run(Thread.java:834)
```

**Comment by qwwdfsad** [comment]
Fixed in 1.5.0
